Tests/SentryTests/SentryScopeSwiftTests.swift

```diff
@@ -108,6 +108,36 @@ class SentryScopeSwiftTests: XCTestCase {
         XCTAssertNil(actual["transaction"])
         XCTAssertNotNil(actual["breadcrumbs"])
     }
+
+    func testInitWithScope() {
```

Comment:
this test is mostly a direct translation of the ObjC version, but with the addition of a check that calling `cloned.applyTo` has the same effect as calling it on the original scope.

Reply:
Please add some validation for assigning `self.span = scope.span` in the `initWithScope`. So basically if I remove `self.span = scope.span` from `initWithScope` this test should fail.

Reply:
Oops, this was what the `Event` comparison was supposed to validate, but I forgot to assign a span to the scope. Removing that line goes from green -> red as expected now.